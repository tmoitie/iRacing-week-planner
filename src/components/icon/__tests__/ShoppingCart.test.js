import React from 'react';
import { render } from '@testing-library/react';
import { describe, test } from '@jest/globals';
import ShoppingCartIcon from '../ShoppingCartIcon';

describe('components/icon/ShoppingCartIcon', () => {
  test('renders correctly', () => {
    const { container } = render(<ShoppingCartIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
