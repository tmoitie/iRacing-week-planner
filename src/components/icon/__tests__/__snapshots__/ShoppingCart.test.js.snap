// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/icon/ShoppingCartIcon renders correctly 1`] = `
<span
  class="glyphicon glyphicon-shopping-cart"
/>
`;
